### PR TITLE
fix: Fix and rename pg_database_url.sh helper script

### PR DESCRIPTION
**Pull request by paescuj**
# Pull Request Template

## Description

The `pg_database_url.sh` helper script (used in Docker entrypoint) wasn't working anymore because of a small syntax error.
This PR fixes this and also renames the file to have an appropiate extension (`.rb`).

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested locally with own Docker image build.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -121,9 +121,7 @@
     allow do
       origins '*'
       resource '/packs/*', headers: :any, methods: [:get, :options]
-      if ActiveModel::Type::Boolean.new.cast(ENV.fetch('CW_API_ONLY_SERVER', false))
```
Comment (sojan-official) [inline config/environments/production.rb:124]: Was this change intended?

Comment (paescuj) [reply config/environments/production.rb:124]: Oops, sorry! No, this is only for local testing purposes - will remove it in a second!

**Review by vishnu-narayanan — APPROVED** [review]
Thank you for the PR.
